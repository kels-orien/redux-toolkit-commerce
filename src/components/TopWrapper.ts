import styled, {css} from 'styled-components';
import { mobileMax } from './../utils/media';

const TopWrapper = styled.div`
    height: 10vh;
`

export default TopWrapper