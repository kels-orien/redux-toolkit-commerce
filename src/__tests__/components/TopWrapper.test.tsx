import React from 'react'
import renderer from 'react-test-renderer'
import TopWrapper from '../../components/TopWrapper'

test('TopWrapper render correctly', () => {
    const tree = renderer.create(<TopWrapper/>).toJSON();

    expect(tree).toMatchSnapshot()

})