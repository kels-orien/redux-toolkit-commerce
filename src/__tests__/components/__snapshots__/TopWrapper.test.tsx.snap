// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopWrapper render correctly 1`] = `
<div
  className="sc-bdnxRM hThiVe"
/>
`;
